packages/react-devtools-shell/webpack-server.js

```diff
@@ -144,14 +142,8 @@ const app = makeConfig(
     react: resolve(builtModulesDir, 'react'),
     'react-debug-tools': resolve(builtModulesDir, 'react-debug-tools'),
     'react-devtools-feature-flags': resolveFeatureFlags('shell'),
-    'react-dom/client': resolve(
-      builtModulesDir,
-      clientAsSeparateBuild ? 'react-dom/unstable_testing' : 'react-dom/client',
-    ),
-    'react-dom': resolve(
-      builtModulesDir,
-      clientAsSeparateBuild ? 'react-dom' : 'react-dom/unstable_testing',
-    ),
```

Comment:
This has been introduced in https://github.com/facebook/react/pull/28271.

This specific config is actually for DevTools (not for `react-devtools-shell`, which may use legacy versions of React). RDT always using React from source, so we don't need to fork it here.